pom.xml

```diff
@@ -94,8 +94,10 @@
 		<package.group>${project.artifactId}</package.group>
 		<package.install.dir>/usr/share/${project.artifactId}</package.install.dir>
 		<package.user>${project.artifactId}</package.user>
+                <package.userId>102</package.userId>
```

Comment:
I really have no idea how this kind of things are usually done. It seems like we are probably not the first to create packages for RedHat (or any other distro) where we need consistent UID, so there is probably a standard solution to this.

In the end, do whatever seems right to you, you probably know more than me about it...